test.py

```diff
@@ -0,0 +1,121 @@
+class Tourist(object):
+
+    def get_info(self):
+        name = str(input('Enter your name: '))
+        room = int(input('Enter the room you want to book: '))
+        nights = int(input('How many nights?: '))
+        self.name = name
+        self.room = room
+        self.nights = nights
+        self.tourist_info = {'name': self.name,
+                             'room': self.room,
+                             'nights': self.nights}
+
+        return self.tourist_info
+
+
+class Hotel(object):
+    # Cодержит данные о номерах отеля.
+    # Номера комнат, информацию о стоимости номера в сутки,
+    # а также методы изменения стоимости в зависимости от сезона года
+
+    prices_all = {"room_101": 15,
+                  "room_102": 20,
+                  "room_103": 25,
+                  "room_104": 30}
+
+    def change_base_price(self):
+        """изменение стоимости любого из номеров"""
+
+        room = str(input('Enter number of room: '))
+        price = int(input('Enter new price: '))
+
+        for key in self.prices_all.keys():
+            if key[-3:] == room:
+                self.prices_all.update({key: price})
+        return self.prices_all
+
+    def change_summer_price(self):
+        # изменение стоимости всех номеров
+        # на 30% в летний период
+
+        for key, value in self.prices_all.items():
+            new_value = value * 1.3
+            self.prices_all.update({key: new_value})
+        return self.prices_all
+
+    def change_autumn_price(self):
+        # изменение стоимости всех номеров
+        # на 20% в осенний период
+
+        for key, value in self.prices_all.items():
+            new_value = value * 1.2
+            self.prices_all.update({key: new_value})
+        return self.prices_all
+
+    def change_spring_price(self):
+        # изменение стоимости всех номеров
+        # на 10% в весенний период
+
+        for key, value in self.prices_all.items():
+            new_value = value * 1.1
+            self.prices_all.update({key: new_value})
+        return self.prices_all
+
+
+class Booking(Tourist):
```

Comment:
I don't understand link between booking(process) and tourist(person). Why should it inherit tourist? 
I suggest to put tourist instance to booking`__init__(self, tourist)`, inherit it from object.